gen-test-report.py

```diff
@@ -0,0 +1,94 @@
+import os
```

Comment:
boo... can we have it as a PowerShell? It's easy to run a ps1 script; py scripts require python installed.

I also have a similar (though probably simpler) reporting implemented in https://github.com/dotnet/aspire/pull/8199/files#diff-6c5c40be894353da23743fb92b7bc0a5b5172c4c207852ff3d41185608ecc0cd (line:90)

![image](https://github.com/user-attachments/assets/fcedce44-b647-4047-8af3-a85c9a8943c2)

Reply:
This is from a local script that I use. Feel free to add some test summary stuff to the github actions results. It would be useful to see the total test counts.